vwr/driver.py

```diff
@@ -42,6 +52,17 @@ def _connect(self):
         self.waiting = False
         if self.timeout:
             self.listener.settimeout(self.timeout)
+        self.get_setpoint()  # dummy request to check if bath is connected
```

Comment:
This checks if the bath responds with something, but you're not checking if its response makes sense. Let's think through what this request should be.

From [the manual](https://us.vwr.com/assetsvc/asset/en_US/id/14127417/contents):

![screenshot from 2017-08-08 10-26-43](https://user-images.githubusercontent.com/1449833/29079980-3c3fe5b0-7c24-11e7-8ac5-23f142e1def9.png)

Maybe reading alarm or operating status would be better? You could then test if the response is e.g. 0 or 1.

I don't have a strong opinion on this, so you can also just ignore.